pkg/kubelet/kubelet.go

```diff
@@ -432,6 +433,10 @@ func (kl *Kubelet) Run(updates <-chan PodUpdate) {
 	if kl.dockerPuller == nil {
 		kl.dockerPuller = dockertools.NewDockerPuller(kl.dockerClient, kl.pullQPS, kl.pullBurst)
 	}
+	if kl.dockerCache == nil {
+		kl.dockerCache = dockertools.NewDockerCache(kl.dockerClient)
+	}
+	go util.Forever(func() { kl.dockerCache.UpdateContainers() }, 50*time.Millisecond)
```

Comment:
I'm not sure if I understand correctly. I agree that it's probably fine for the first request to have higher latency. But assuming "cache with timeout" how would you like to refresh it (not doing it periodically)? 

Also - I don't think this has any significant overhead on Kubelet side (when idle), although I agree it has an effect on Docker. But maybe it's enough to increase the frequency of this calls? E.g. maybe increasing it to 1 second of 5 seconds is enough?